app/assets/javascripts/discourse/components/topic-list-item.js.es6

```diff
@@ -35,6 +35,38 @@ export const ListItemDefaults = {
   attributeBindings: ["data-topic-id"],
   "data-topic-id": Ember.computed.alias("topic.id"),
 
+  didInsertElement() {
+    if (typeof this.get("topic.read_by_group_member") !== "undefined") {
```

Comment:
You should have a `this._super(...arguments)` in here and `willDestroyElements` just to be safe.